documentation/content/el/books/handbook/desktop/_index.adoc

```diff
@@ -408,7 +408,7 @@ endif::[]
 
 [source,shell]
 ....
-# pkg_add -r openoffice.org
+# pkg install openoffice.org
```

Comment:
Existing ports include: 

```
editors/openoffice-4
editors/openoffice-devel
textproc/p5-OpenOffice-OODoc
```

Reply:
editors/libreoffice is probably the better maintained of the two forks